index.html

```diff
@@ -7,7 +7,7 @@
     <meta name="Description" content="A Simple To Do List WebApp.">
 
     <!-- Google Font: Quick Sand -->
-    <link href="https://fonts.googleapis.com/css2?family=Work+Sans:wght@300&display=swap" rel="stylesheet"> 
+    <link href="https://fonts.googleapis.com/css2?family=Work+Sans:wght@300;600&display=swap" rel="stylesheet"> 
```

Comment:
- In reference to what I mentioned earlier, 300 size looks readable and pleasing, so 600 might not be needed.

![Screenshot (764)](https://user-images.githubusercontent.com/61280281/98966609-7b77b500-2531-11eb-8a28-a32acc6080c8.png)

![Screenshot (765)](https://user-images.githubusercontent.com/61280281/98966601-7a468800-2531-11eb-8689-22cb3614a088.png)